db/schema.rb

```diff
@@ -159,6 +158,17 @@
     t.string   "teacher_home_path",                                  :default => "/getting_started"
   end
 
+  create_table "admin_settings_vendor_interfaces", :force => true do |t|
```

Comment:
@pjanik looks like you haven't run the migration to remove these probe tables. That migration should be in master.